Dockerfile

```diff
@@ -28,31 +26,33 @@ RUN set -ex \
 	&& node --version \
 	&& npm --version \
 	&& yarn --version \
-	&& bash -c "${DOCKER_FILE_INSTALL_COMMAND:?Build argument DOCKER_FILE_INSTALL_COMMAND needs to be set (check READEME.md)!}" \
-	&& echo "OK"
+	&& python3 -m pip install 'psycopg2-binary==2.9.4' && python3 -m pip install 'Django==4.1.2' \
+	&& npm i -g turbo \
+    && echo "OK"
 
 # Build container
 FROM base AS build
-ARG DOCKER_FILE_BUILD_COMMAND
-ENV DOCKER_FILE_BUILD_COMMAND ${DOCKER_FILE_BUILD_COMMAND}
 USER app:app
-WORKDIR /home/app
-RUN echo "# Build time .env config!" >> /home/app/.env && \
-	echo "COOKIE_SECRET=undefined" >> /home/app/.env && \
-	echo "DATABASE_URL=undefined" >> /home/app/.env && \
-	echo "REDIS_URL=undefined" >> /home/app/.env && \
-	echo "NODE_ENV=production" >> /home/app/.env
-# If you don't have this directory, you can create it by command `bash ./bin/warm-docker-cache` or just create empty ./.docker-cache-warming dir (no cache mode)
-ADD --chown=app:app ./.docker-cache-warming /home/app
-# Cache packages!
-RUN set -ex && npm i -g turbo && yarn install --immutable
-ADD --chown=app:app . /home/app
-RUN set -ex && yarn && bash -c "${DOCKER_FILE_BUILD_COMMAND:?Build argument DOCKER_FILE_BUILD_COMMAND needs to be set (check READEME.md)!}" && \
-    yarn cache clean && rm -rf /home/app/.env && rm -rf /home/app/.config && rm -rf /home/app/.yarn && rm -rf /home/app/.cache && \
-    ls -lah /home/app/
+WORKDIR /app
+
+RUN echo "# Build time .env config!" >> /app/.env && \
+	echo "COOKIE_SECRET=undefined" >> /app/.env && \
+	echo "DATABASE_URL=undefined" >> /app/.env && \
+	echo "REDIS_URL=undefined" >> /app/.env && \
+	echo "NODE_ENV=production" >> /app/.env
+
+ADD --chown=app:app . /app
```

Comment:
IMO there is no reason to use `ADD` here. If you're working with local non compressed files and dirs -> use `COPY` as you did before and after that layer